layouts/partials/header.html

```diff
@@ -0,0 +1,59 @@
+<!-- Navbar Start -->
+<header id="topnav">
+    <div class="container d-flex justify-content-between align-items-center">
+        <!-- Logo container-->
+        <a class="logo" aria-label="Home" href='{{ relLangURL "" }}'>
+            {{ with resources.Get "images/logos/mark.svg" }}
+                {{ .Content | safeHTML }}
+            {{ end }}
+        </a>
+        <!-- End Logo container-->
+
+        <div class="d-flex align-items-center">
+
+            <div id="navigation">
+                <!-- Navigation Menu -->
+                <ul class="navigation-menu nav-right">
+                    {{- range .Site.Menus.primary }}
+                    <li><a href="{{ relLangURL .URL }}">{{ .Name }}</a></li>
+                    {{ end }}
+                </ul><!--end navigation menu-->
+            </div><!--end navigation-->
+
+            <!-- Social Links Start -->
+            {{ with $.Scratch.Get "social_list" }}
+            <ul class="social-link d-flex list-inline mb-0">
+                {{ range . }}
+                    {{ $path := printf "images/social/%s.%s" . "svg" }}
+                    <li class="list-inline-item mb-0">
+                        <a href="{{ index site.Params.social . | safeURL }}" alt="{{ . }}" rel="noopener noreferrer" target="_blank">
```

Comment:
This was the issue. The template forced you to use provide a path (and append the path to a `.com` which didn't work for GitHub links) instead of a full URL. I've changed it to accept a full URL.